src/screenshot/screenshot.c

```diff
@@ -35,17 +35,78 @@ static bool exec_screenshooter(const char *path) {
 
 	return stat == 0;
 }
+static bool exec_screenshooter_interactive(const char *path) {
+	pid_t pid = fork();
+	if (pid < 0) {
+		perror("fork");
+		return false;
+	} else if (pid == 0) {
+		char cmd[strlen(path) + 25];
+		snprintf(cmd, sizeof(cmd), "grim -g \"$(slurp)\" -- %s", path);
```

Comment:
Note: this will fall apart if the path contains spaces or other special characters, but doesn't matter for now because the path is hardcoded.

In the future, we may want to set an env variable containing the path, and use `sh`'s variable expansion to ensure there are no quoting shenanigans.